index.html

```diff
@@ -22,6 +22,12 @@ <h2>Tracker Reporting</h2>
 	<li><a href="./tracker-reporting/1major-via-fetch.html">1 major tracker loaded via fetch</a> <a href="./tracker-reporting/1major-via-fetch.html?delay=5000">[Delay 5s]</a></li>
 </ul>
 
+<h2>Ad flow</h2>
+
+<ul>
+	<li><a href="./adClickFlow/serp/index.html">Ad flow</a></li>
+</ul>
```

Comment:
This doesn't really explain what this test does. We don't provide any additional info in `serp/index.html` either. I think we should leave at least a bit of explanation.